Validation/RecoParticleFlow/plugins/PFAnalysis.cc

```diff
@@ -73,40 +73,50 @@ class ElementWithIndex {
       : orig(_orig), idx_block(_idx_block), idx_elem(_idx_elem){};
 };
 
-int find_element_ref(const vector<ElementWithIndex>& vec, const edm::RefToBase<reco::Track>& r) {
+vector<int> find_element_ref(const vector<ElementWithIndex>& vec, const edm::RefToBase<reco::Track>& r) {
+  vector<int> ret;
   for (unsigned int i = 0; i < vec.size(); i++) {
     const auto& elem = vec.at(i);
     if (elem.orig.type() == reco::PFBlockElement::TRACK) {
       const auto& ref = elem.orig.trackRef();
-      assert(ref.isNonnull());
-      if (ref.key() == r.key()) {
-        return i;
-      }
-    } else if (elem.orig.type() == reco::PFBlockElement::BREM) {
-      const auto& ref = elem.orig.trackRefPF();
-      if (ref.isNonnull()) {
-        const auto& ref2 = ref->trackRef();
-        assert(ref2.isNonnull());
-        if (ref2.key() == r.key()) {
-          return i;
+      if (ref.isNonnull() && ref->extra().isNonnull()) {
+        if (ref.key() == r.key()) {
+          ret.push_back(i);
         }
       }
     }
-  }
-  return -1;
+    //this doesn't seem to work consistently: BREM and GSF sometimes get wrongly associated
```

Comment:
is this commented-out code expected to be fixed and uncommented in the future? if not, delete it